Update Enrich_URLs_and_Domains workflow with indicator extraction (#301)
rapid7/insightconnect-workflows: workflows/Enrich_URLs_and_Domains_with_VirusTotal_from_Microsoft_Teams/help.md

# Description

Lookup and enrich suspicious URLs and domains with a simple Microsoft Teams command. URL and domain analysis provided by VirusTotal is returned in a Microsoft Teams thread.

> More than one URL or domain can be specified in a single command.

Reviewer: ```suggestion
Multiple URLs or domains can be specified in a single command.
```
